client/landing/stepper/declarative-flow/migration/index.tsx

```diff
@@ -162,13 +174,14 @@ const useCreateStepHandlers = ( navigate: Navigate< StepperStep[] >, flowObject:
 				const siteSlug = getFromPropsOrUrl( 'siteSlug', props ) as string;
 
 				const plan = props?.plan as string;
-				const backToStep = {
-					step: MIGRATION_UPGRADE_PLAN.slug,
-					flow: flowPath,
-				};
 
 				if ( props?.action === MigrationUpgradePlanActions.IMPORT_CONTENT_ONLY ) {
-					return goToImporter( 'importerWordpress', siteId, siteSlug, backToStep );
+					return goToImporter( {
```

Comment:
Not sure if it should be part of this issue. We can definitely work on this in another PR.

I think we should use the redirect with `replace` only in this situation.

The reason is that if I navigate to  `/setup/migration` and click on `Substack`, the browser back doesn't work. Make sense?

Reply:
👍 x10
I will open a new PR